skynet/modules/stt/streaming_whisper/state.py

```diff
@@ -45,44 +43,25 @@ def __init__(
         # silence count before starting to drop incoming silent chunks
         self.silence_count_before_ignore = add_max_silent_chunks
         self.final_after_x_silent_chunks = final_after_x_silent_chunks
-        self.force_final_duration_threshold = force_final_duration_threshold
-        self.perform_final_after_silent_seconds = perform_final_after_silent_seconds
         self.uuid = utils.Uuid7()
         self.transcription_id = str(self.uuid.get())
+        self.last_received_chunk = utils.now()
```

Comment:
Doesn't it make sense to initialize this to None and set it to an actual timestamp the moment we process the first chunk?

Reply:
It's never `None`, as we set it to `utils.now` in the State constructor when the participant receives the first chunk https://github.com/jitsi/skynet/blob/aa6c04458056b3d716338ec29838c50729a1221b/skynet/modules/stt/streaming_whisper/meeting_connection.py#L58

Reply:
👍